keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ConvertExampleFilesTest.kt

```diff
@@ -17,47 +17,32 @@
  */
 package com.netflix.spinnaker.keel.rest
 
-import com.fasterxml.jackson.databind.jsontype.NamedType
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper
 import com.fasterxml.jackson.module.kotlin.readValue
 import com.netflix.spinnaker.keel.api.ec2.ApplicationLoadBalancerSpec
 import com.netflix.spinnaker.keel.api.ec2.ClassicLoadBalancerSpec
 import com.netflix.spinnaker.keel.api.ec2.ClusterSpec
-import com.netflix.spinnaker.keel.api.ec2.EC2_APPLICATION_LOAD_BALANCER_V1_1
-import com.netflix.spinnaker.keel.api.ec2.EC2_CLASSIC_LOAD_BALANCER_V1
-import com.netflix.spinnaker.keel.api.ec2.EC2_CLUSTER_V1
-import com.netflix.spinnaker.keel.api.ec2.EC2_CLUSTER_V1_1
-import com.netflix.spinnaker.keel.api.ec2.EC2_SECURITY_GROUP_V1
 import com.netflix.spinnaker.keel.api.ec2.SecurityGroupSpec
 import com.netflix.spinnaker.keel.api.ec2.old.ClusterV1Spec
-import com.netflix.spinnaker.keel.api.plugins.SupportedKind
+import com.netflix.spinnaker.keel.api.titus.TestContainerVerification
 import com.netflix.spinnaker.keel.api.titus.TitusClusterSpec
+import com.netflix.spinnaker.keel.api.titus.TitusServerGroup.Location
 import com.netflix.spinnaker.keel.core.api.SubmittedDeliveryConfig
-import com.netflix.spinnaker.keel.ec2.jackson.registerKeelEc2ApiModule
-import com.netflix.spinnaker.keel.test.configuredTestYamlMapper
-import com.netflix.spinnaker.keel.titus.TITUS_CLUSTER_V1
-import com.netflix.spinnaker.keel.titus.jackson.registerKeelTitusApiModule
 import dev.minutest.junit.JUnit5Minutests
 import dev.minutest.rootContext
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
 import strikt.api.expectCatching
 import strikt.assertions.isA
+import strikt.assertions.isEqualTo
 import strikt.assertions.isSuccess
 
+@SpringBootTest
```

Comment:
```suggestion
@SpringBootTest(webEnvironment = NONE)
```

Probably want this to avoid it starting up the embedded web server since the tests don't go via the REST API.

Reply:
Unfortunately, I still don't have a good enough mental model about how Spring's DI implementation works to understand why we don't need to specify the main class. All I know is that it seems to work.